charts/fluent-bit/templates/_pod.tpl

```diff
@@ -37,9 +37,19 @@ containers:
   {{- end }}
     image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
     imagePullPolicy: {{ .Values.image.pullPolicy }}
-  {{- if .Values.env }}
+  {{- if or .Values.env .Values.envWithTpl }}
     env:
+      {{- if .Values.env }}
```

Comment:
Using `with` here would be better than `if`.

Reply:
Done